code/game/objects/items/stacks/medical.dm

```diff
@@ -59,7 +59,7 @@
 
 /// Apply the actual effects of the healing if it's a simple animal, goes to [/obj/item/stack/medical/proc/heal_carbon] if it's a carbon, returns TRUE if it works, FALSE if it doesn't
 /obj/item/stack/medical/proc/heal(mob/living/patient, mob/user)
-	if(patient.stat == DEAD)
+	if(isanimal(patient) && patient.stat == DEAD) // SKYRAT EDIT CHANGE, ORIGINAL DID NOT CHECK isanimal -- CARBON CHECKS ARE DONE IN heal_carbon()
```

Comment:
```suggestion
	if(isanimal(patient) || patient.stat == DEAD) // SKYRAT EDIT CHANGE, ORIGINAL DID NOT CHECK isanimal -- CARBON CHECKS ARE DONE IN heal_carbon()
```

Reply:
This then won't do anything, this check will automatically fail if the patient is dead and heal() will return without calling heal_carbon()